tests/Support/Module.php

```diff
@@ -0,0 +1,555 @@
+<?php
+
+namespace Tests\Psalm\LaravelPlugin\Support;
+
+use Codeception\Exception\ModuleRequireException;
+use Codeception\Exception\TestRuntimeException;
+use Codeception\Module as BaseModule;
+use Codeception\Module\Cli;
+use Codeception\Module\Filesystem;
+use Codeception\TestInterface;
+use Composer\InstalledVersions;
+use Composer\Semver\Semver;
+use Composer\Semver\VersionParser;
+use PackageVersions\Versions;
+use PHPUnit\Framework\Assert;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use OutOfBoundsException;
+use PHPUnit\Framework\SkippedTestError;
+use RuntimeException;
+
+/**
+ * Duplicate of https://github.com/psalm/codeception-psalm-module
```

Comment:
Hey could you help me understand why we need this duplicated? Do we need some changes merged upstream to `codeception-psalm-module`? @weirdan has been great at merging prs fast! Let me know

Reply:
It's duplicated because of the changes to codeception. Doing the PR against https://github.com/psalm/codeception-psalm-module would require a newer version of codeception which doesnt support PHP 7.x

Reply:
I just went through https://github.com/psalm/codeception-psalm-module and unfortunately even checking it out and running the tests there without doing any code modifications there are *13* unit test failures.

So someone would first have to go fix those errors. Then do the work for codeception 5.